pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -331,14 +331,14 @@ func TestReconcileProvisionOK(t *testing.T) {
 			AssertThatNamespace(t, username+"-code", r.client).
 				HasNoOwnerReference().
 				HasLabel("toolchain.dev.openshift.com/owner", username).
-				HasLabel("toolchain.dev.openshift.com/type", "code")
+				HasLabel("toolchain.dev.openshift.com/type", "code").
+				HasNoLabel("toolchain.dev.openshift.com/revision")
 		})
 
 		t.Run("inner resources created for existing namespace", func(t *testing.T) {
```

Comment:
and could we verify the existence of the labels when the inner resources are created:
```go
			AssertThatNamespace(t, username+"-dev", fakeClient).
				HasResource("user-edit", &authv1.RoleBinding{}).
				HasLabel("toolchain.dev.openshift.com/owner", username).
				HasLabel("toolchain.dev.openshift.com/type", "dev").
				HasLabel("toolchain.dev.openshift.com/revision", "abcde11").
				HasLabel("toolchain.dev.openshift.com/tier", "basic")
```